content/docs/dvclive/how-to/checkpoints.md

```diff
@@ -0,0 +1,42 @@
+# Integrate with DVC Checkpoints
```

Comment:
Maybe make this more general ("Resume training"?) in case people don't know about DVC checkpoints or are using `resume` without checkpoints.